### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by prithviBytes**
Hi team,
Please review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# ideal-octo-doodle
\ No newline at end of file
+# ideal-octo-doodle
+
+Code done
```
Comment (hntejas) [inline README.md:3]: Code done 1 would perform better

**Review by hntejas — CHANGES_REQUESTED** [review]
Please check the review

File: README.md (hunk 1 above)
Comment (prithviBytes) [reply README.md]: could you check it.

**Review by hntejas — APPROVED** [review]
looks good

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# ideal-octo-doodle
\ No newline at end of file
+# ideal-octo-doodle
+
+Code done 1
```
Comment (themohammadsa) [inline README.md:3]: Change this one

**Review by themohammadsa — CHANGES_REQUESTED** [review]
Change it

**Review by themohammadsa — APPROVED** [review]
Approved it

**Pull request merged**